zone/trading.cpp

```
@@ -2812,24 +2814,184 @@ void Client::SendTraderMode(BazaarTraderBarterActions status)
+				(update_matching_items && !item->IsStackable() && item->GetID() == selected_item_id) ||
+				(!update_matching_items && item->GetUniqueID() == selected_unique_id)
+			) {
+				items.push_back(item);
+			}
+		}
+	}
+
+	return items;
+}
+
+static TraderRepository::Trader BuildTraderItemRow(
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)